nsv13/code/modules/overmap/weapons/mines.dm

```diff
@@ -0,0 +1,99 @@
+/obj/structure/space_mine
+	name = "space mine"
+	desc = "Like a naval mine, but in space!"
+	icon = 'nsv13/icons/overmap/effects.dmi'
+	icon_state = "mine_syndicate"
+	anchored = TRUE
+	density = FALSE
+	layer = ABOVE_MOB_LAYER
+	animate_movement = NO_STEPS
+	max_integrity = 300
+	integrity_failure = 100
+	var/datum/star_system/current_system
+	var/faction = "syndicate" //evil mines
+	var/damage = 100
+	var/damage_type = BRUTE
+	var/damage_flag = "overmap_heavy"
+	alpha = 110 //They're supposed to be sneaky, their main advantage is being cloaked
+
+/obj/structure/space_mine/Initialize(mapload, var/new_faction, var/datum/star_system/system, var/list/coordinates)
```

Comment:
We do have a proc to get the center turf of a overmap (`get_center()`), you could just use pass that as first arg and avoid another init var, right?

Reply:
True, it just won't provide pixel accuracy. I guess it doesn't matter too much